config/knexfile.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     client: 'pg',
     connection: {
       connectionString: process.env.DATABASE_URL,
-      ssl: true,
+      ssl: { rejectUnauthorized: false },
```

Comment:
Was this what was affecting the push to Heroku?

Reply:
Yes, this and along with the fact that Heroku was using a different Database URL beside the one we were originally referencing.